### ISSUE TITLE
Fix paddings respecting the safe area on iPhone X & later

### ISSUE THREAD

**Issue by airatk** [firasdib/Regex101#2156]
## Bug Description
Some parts of the PWA UI of Regex101 are left out of the safe area of iPhone X & later. See the screenshots:

<div>
    <img src='https://github.com/firasdib/Regex101/assets/19852442/40859998-9c2d-41a8-94ee-fa28c8ddc68b' width= '240' />
    <img src='https://github.com/firasdib/Regex101/assets/19852442/8452fe3b-cecf-4e95-95e6-a5830294ad2e' width= '240' />
    <img src='https://github.com/firasdib/Regex101/assets/19852442/d12e6a8e-7709-41a4-b31a-dc130199332c' width= '240' />
</div>

### Reproduction steps
This is reproducible on any iPhone X & later by adding Regex101 as a bookmark to the home screen (in other words, using Regex101 as PWA). 

### Expected Outcome
The edges of the PWA UI should look like demos in this post: https://webkit.org/blog/7929/designing-websites-for-iphone-x/
In other words, the safe area of mentioned devices should be taken into account. 

### Browser & OS
It affects iPhone X & later, no matter what browser and what version of iOS is used to add Regex101 as a bookmark on the home screen.

**Comment by firasdib** [comment]
Thank you for this, I have implemented fixes for this and will have them released soon.
